### DISCUSSION TITLE
How to run faster-whisper from Windows powershell?

### DISCUSSION THREAD

**Discussion by ghost** [SYSTRAN/faster-whisper#146] [Q&A] [answered]
Hi, sorry, maybe this is a really dumb question, but I can't figure out how to actually call up and run faster-whisper. I can run regular whisper just fine from Powershell (W10 x64), but after PIP installing faster-whisper there's no corresponding script/exe that shows up in my virtual environment's "scripts" folder, whether I use Python 3.10 or Python 3.11. How do you actually start it up?

**Comment by guillaumekln** [comment, marked as answer]
See https://github.com/jordimas/whisper-ctranslate2 which is a command line utility built on top of faster-whisper.

**Comment by Purfview** [comment]
Another alternative is https://github.com/Purfview/whisper-standalone-win with modified cli mostly taken from OpenAI's repo.
